src/model/mod.rs

```diff
@@ -249,41 +251,106 @@ impl Stats {
     pub fn get_agility(&self) -> usize {
         self.agility
     }
-}
 
-/// Struct containing all the information about a single Modifier for a Model
-///
-/// ## Attributes
-/// stat (Stats): Stat modified by the Modifier
-///
-/// nb_dice (usize): The number of dice
-///
-/// requirements (Vec<String>): The requirements for the Modifier to apply (as an array of flags)
-#[derive(Clone, PartialEq, Eq, Debug)]
-pub struct Modifier {
-    stat: Stats,
-    bonus: bool,
-    nb_dice: usize,
-    requirements: Vec<String>,
-}
+    /// Buff the Stats with a GlobalModifier
+    ///
+    /// ## Parameters
+    /// &modifier (GlobalModifier): The GlobalModifier to buff the Stats with
+    ///
+    /// ## Returns
+    /// &Stats: The Stats buffed
+    fn buff_global_modifier(&mut self, modifier: &modifier::GlobalModifier) -> &Stats {
```

Comment:
This function name is misleading I think people will believe it will buf the global modifier but instead it buff the stats.
And why do we need the return value ? Just change the Stats maybe ?

Reply:
Changed it :heavy_check_mark: